main.cpp

```diff
@@ -13,75 +13,606 @@ What you create in this project will be used as the basis of Project 5 in the co
+    3) number of people working in store (int)
+    4) amount of utilities (float)
+    5) daily income (float)
+3 things it can do:
+    1) Sell daily use products
+    2) Help customers with any issue
+    3) Deliver products
 */
+
+struct Market
+{
+    //number of products in local inventory
+    float numProdLocalInv = 215.3f;
+    //number of vehicles used in logistics
+    int numLogisticVehicles = 5; 
+    //number of people working in store
+    int numPeopleWorkingAtStore =32; 
+    //amount of utilities
+    float basicUtilities = 320.53f;  
+    //daily income
+    float dailyIncome = 2750; 
+
+    struct Customer
+    {
+        int age = 38;
+        bool member = true;
+        int visitsPerWeek = 7;
+        std::string gender = "Male";
+        std::string transportation = "Scooter";
+
+        void eatFood(int Frequency = 5);
+        float calculateWorkedTime(bool atHome = true, 
+                                  float time = 8.5f); 
+        float paymonthlyShould(float rent = 1500,  
+        float food = 450, 
+        float fun = 120,
+        float other = 230);
+    };
+
+    //Sell daily use products
+    void sellProducts(Customer Nick); 
+    //Help customers with any issue
+    void solveCustomerNeeds(Customer Jack, bool solved = true, bool quest = false); 
+    //Deliver products
+    void deliverProducts(Customer Phil, bool success = true, int time = 35);
+};
+
+/*
+Thing 2) University
+5 properties:
+    1) number of classrooms (int)
+    2) number of laboratories (int)
+    3) number of professors (int)
+    4) amount of semestral income (float)
+    5) classes taken per period (int)
+3 things it can do:
+    1) Teach students
+    2) Do collaborate research 
+    3) Perform cultural activities
+ */
+
+struct University
+{
+    //number of classrooms
+    int numClassrooms = 80; 
+    //number of laboratories
+    int numLabs = 36; 
+    //number of professors
+    int numProfessors = 95; 
+    //amount of semestral income
+    float semIncome = 5'000'000; 
+    //classes taken per period
+    int classesPerSemester = 8362; 
+
+    struct Professor
+    {
+        int age = 58;
+        std::string profession = "Engineer";
+        std::string postgraduateStudies = "MsC";
+        std::string teachingRank = "Band 4";
+        int yearsExperience = 15;
+
+        float work(int research = 28, int teaching = 16);
+        float monthlyPay(float rent = 1800, 
+                         float food = 550, 
+                         float fun = 120,
+                         float other = 150);
+    };
+
+    struct Student
+    {
+        int age = 18;
+        std::string department = "Engineer";
+        std::string career = "AgroIndustrial Engineer";
+        std::string hobby = "Play Bass";
+        int semestralCredits = 9;
+
+        float weekstudyTime(int research = 28, int teaching = 16);
+        float monthlyExpenses(float food = 250, 
+        float fun = 220,
+        float other = 350);
```

Comment:
`monthlyExpenses` is not an action
pick a function name and use a verb in the name you pick
ex: `computeMonthlyExpenses`
- [ ] align the function parameters vertically with the opening `(`
- [ ] revise